Dockerfile.tests

```diff
@@ -0,0 +1,16 @@
+FROM python:3.8.1
```

Comment:
Please move dockerfiles for test purposes to the tests directory. Please use the same setup as in Dockerfile for development (base image, building steps ...). Maybe the dev dockerfile can just be used for tests.

Reply:
The next version has a different dockerfile integration.